tests/utils.go

```diff
@@ -2437,3 +2437,13 @@ func RenderTargetcliPod(name, disksPVC string) *k8sv1.Pod {
 		},
 	}
 }
+
+func CheckResultShellCommandOnVmi(vmi *v1.VirtualMachineInstance, cmd, output string, timeout int) bool {
+	res, err := console.SafeExpectBatchWithResponse(vmi, []expect.Batcher{
+		&expect.BSnd{S: fmt.Sprintf("%s\n", cmd)},
+		&expect.BExp{R: console.PromptExpression},
+	}, timeout)
+	Expect(err).ToNot(HaveOccurred())
+	By(fmt.Sprintf("Checking for \ncommand: '%s'\noutput: '%v'\ncontaining: '%s'", cmd, res[0].Output, output))
+	return strings.Contains(res[0].Output, output)
```

Comment:
nit:
```suggestion
	ExpectWithOffset(1, res[0].Output).To(ContainSubstring(output))
```
I think in such way it should print pretty much the same message as in `By(...)` here and the function itself can return void.

Reply:
Additionally, I would also add
```go
Expect(res).ToNot(BeEmpty())
```
before dereferencing `res[0]`